.github/docs/README.md

```diff
@@ -13,6 +13,65 @@ This documentation is a work in progress and will be updated as new features are
 | **BaseFeePerGas** | The minimum gas price required for a transaction to be included in the current block.|
 
 
+## Methodology
+
+### Before Galactica
+
+The pre-Galactica transaction model implements a fixed gas pricing mechanism with the following characteristics:
+
+1. **Limited Gas Price Range**: Legacy transactions operate within a constrained gas price range
+2. **Fixed Validator Tip**: Validators receive a fixed 30% of the total gas fee paid by users
+3. **Base Gas Price**: A network parameter (currently set at 1e13 wei) that can be adjusted by the steering committee
```

Comment:
The base gas price is `1e15` https://github.com/vechain/thor/blob/release/galactica/thor/params.go#L62

Reply:
Initial value is `1e15` and we have changed it to `1e13` through a governance vote. https://vevote.vechain.org/stakeholdersDetail?voting_id=0xa6416a72f816d3a69f33d0814700545c8e3fe4be.6